package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "test": "turbo run testing",
     "elm-format": "elm-format --validate ast-codec init-templates new-package parseElm review template",
-    "eslint-check": "eslint . --report-unused-disable-directives",
+    "eslint-check": "eslint . --report-unused-disable-directives --max-warnings=0",
```

Comment:
I set a few, and the default configs do a few as well.
I'd mostly like to set it so it looks different in the editor. Though the ESLint integration in VSCode is acting up.

Reply:
Alright, that's fine by me, seeing it as different in the editor is the one place where I agree it can make sense to have rules marked as errors or warnings :smile: